gcn_classic_to_json/notices/CALET_GBM_FLT_LC/__init__.py

```diff
@@ -0,0 +1,40 @@
+from astropy.time import Time
+import struct
+
+# TDJ to JD offest
+TJD0 = 2440000.5
+
+# Will have to rename schema after the location is decided
+
+def parse(bin_arr):
+    # e_high and e_low are both encoded in the same 4 byte quantity (2 bytes each)
+    # we use bit-masking and bit-shifting to extract the values from the 4 byte integer
+    image_energy_range = [int(bin_arr[17] & 0xFFFF), int(bin_arr[17] >> 16)]
+    # similarly we do this with lat and long
+    lat, lon = (bin_arr[16] >> 16), bin_arr[16] & 0xFFFF
+    # similarly using bit-masking to get detector status
+    # I have defined 0 in the binary as on and 1 as triggered
+    detector_options = ["on", "triggered"]
+    detectors_status = [int(bin_arr[19] & 0x00000001), int((bin_arr[19] >> 1) & 0x00000001) , int((bin_arr[19] >> 2) & 0x00000001)]
+    detectors = {"HXM1" : detector_options[detectors_status[0]],
+                 "HXM2" : detector_options[detectors_status[1]],
+                 "SGM" : detector_options[detectors_status[2]]}
+    return {
+        # "$schema" : "",
+        "mission" : "CALET",
+        "instrument" : "GBM",
+        # "alert_type" : "initial", # does not exist in the binaries
+        # "record_number" = 1, # does not exist in binaries
+        "id" : [int(bin_arr[4])],
+        "messenger" : "EM",
+        "trigger_time" : Time(bin_arr[5]+(bin_arr[6]/8640000)+TJD0, format="jd").isot+"Z",
```

Comment:
This code is repeated in multiple modules now. Please refactor into a common module --- a good name might be gcn_classic_to_json/utils.py.